src/lcf/third_party/string_view.hpp

```diff
@@ -0,0 +1,1545 @@
+// Copyright 2017-2020 by Martin Moene
+//
+// string-view lite, a C++17-like string_view for C++98 and later.
+// For more information see https://github.com/martinmoene/string-view-lite
+//
+// Distributed under the Boost Software License, Version 1.0.
```

Comment:
I'd suggest mentioning it on README.md as we do with inih.

Reply:
fixed